sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveExternalCatalogSuite.scala

```diff
@@ -71,4 +72,19 @@ class HiveExternalCatalogSuite extends ExternalCatalogSuite {
     assert(!rawTable.properties.contains(HiveExternalCatalog.DATASOURCE_PROVIDER))
     assert(DDLUtils.isHiveTable(externalCatalog.getTable("db1", "hive_tbl")))
   }
+
+  test("alter table schema should save case-sensitive schema to properties") {
```

Comment:
This test checks that the case-sensitive schema is written to Hive as table properties when using ```HiveExternalCatalog```